src/lib/units/year.js

```diff
@@ -36,6 +48,9 @@ addParseToken('YYYY', function (input, array) {
 addParseToken('YY', function (input, array) {
     array[YEAR] = hooks.parseTwoDigitYear(input);
 });
+addParseToken('Y', function (input, array) {
+    array[YEAR] = parseInt(input, 10);
```

Comment:
Anything wrong with `input | 0`, or are we wanting to be verbose and let ES6->ES5 transpilation do those tricks?  (I suppose same for above comment)